src/ripple/app/misc/impl/TxQ.cpp

```diff
@@ -1492,27 +1497,19 @@ TxQ::doRPC(Application& app) const
     levels[jss::median_level] = to_string(metrics.medFeeLevel);
     levels[jss::open_ledger_level] = to_string(metrics.openLedgerFeeLevel);
 
-    auto const baseFee = view->fees().base;
+    auto const& baseFee = view->fees().base;
```

Comment:
Nit: XRPAmount is small, taking it by value is fine (no change needed).

Reply:
Fixed anyway